### DISCUSSION TITLE
No 6.5.1 in Firefox AMO?

### DISCUSSION THREAD

**Discussion by segevfiner** [vuejs/devtools-v6#2116] [Q&A]
Just making sure you didn't forget to submit it and it's just stuck in review or something else like that. @Akryum

**Comment by Akryum** [comment]
It's probably still in review, will check. 👀

**Reply by Sirs0ri** [reply]
I noticed that 2 weeks after the release of 6.6.0 and .1, those versions still aren't available on Firefox - is it a common issue for Addons to be reviewed for this long?

**Comment by erropix** [comment]
@Akryum It's the same for 6.6.1 now, the releases were 3 weeks ago, but the Firefox add-on is outdated.

**Reply by Akryum** [reply]
Still in review

**Reply by segevfiner** [reply]
Maybe someone should ping their support about this at this point...? Though I will assume there might just be something wrong in AMO or Mozilla at this point...

**Reply by segevfiner** [reply]
@Akryum Did you contact their support? I won't be surprised if there was some outage and it will just remain stuck forever otherwise...

**Reply by segevfiner** [reply]
I have seen other extensions updated by now... Probably needs to be resubmitted or to contact support...

**Reply by segevfiner** [reply]
@Akryum Could you try resubmitting it?

**Reply by Sirs0ri** [reply]
6.6.3 finally showed up for me a few days ago :)
